frog/imports/ui/editor/Editor.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react';
+import Activity from './Activity.jsx';
+import { Activities } from '../../api/activities.js';
+
+export default class Editor extends Component {
+
+  //For now, we show the result of the form with an alert, but of course it'll be then added to the database.
+  handleSubmit() {
+    if(!this.refs.newActivity.haveFieldsCompleted()) {
+      alert("Not all fields have been filled !");
+    }
+    else if(!this.refs.newActivity.areAnswersInChoices()){
+      alert("Not all answers are in the choices !");
+    }
+    else {
+      var item = this.refs.newActivity.generateActivity();
+      Activities.remove(item['_id']);
```

Comment:
Activities should not be removed, when we want to write an activity with the same `_id`. Instead, prompt the user to chose an other `_id`. I did it like this in the Admin section because it uses the JSON files and nothing will be lost, but it is not the behaviour that we want here. It could lead to teacher losing their activities.